ch2driver/pytpcc/drivers/nestcollectionsdriver.py

```diff
@@ -557,7 +790,7 @@ def __init__(self, ddl, clientId, TAFlag="T",
                 cert_reqs="CERT_NONE",
                 socket_options=[  # Set TCP keep-alive options for long running analytics queries
                     (socket.SOL_SOCKET, socket.SO_KEEPALIVE, 1),
-                    (socket.IPPROTO_TCP, socket.TCP_KEEPIDLE, 120),
+#                    (socket.IPPROTO_TCP, socket.TCP_KEEPIDLE, 120),
```

Comment:
Was this setting causing a problem? I'm mainly curious - removing the TCP_KEEPIDLE setting by itself probably won't hurt.

I originally enabled TCP_KEEPALIVE to prevent connections hanging when doing very long (1h+) queries (on lots of data). Sorry if there was some collateral damage

Reply:
It was causing a problem on my laptop, so removed it for now. Please add it back if you think right, not my area of expertise.

Reply:
Ah ok in that case I think we can remove this one - sorry it was causing trouble with your laptop, I have to say I didn't specifically test it on my laptop I was just using perf machines. The other options (particularly the SO_KEEPALIVE option) should be kept though.